docs/mine/spacerace.md

```diff
@@ -15,7 +15,7 @@ A miner’s “location,” for regional leaderboards, is the physical location
 
 ## How do I participate?
 
-**The calibration period is currently open.** To join, run at least 1 miner on the [calibration devnet](https://docs.filecoin.io/how-to/networks/#calibration-devnet). A preview of calibration standings is available on the [Preview Dashboard](https://calibration.spacerace.filecoin.io/). 
+To join, run at least 1 miner on the [testnet](https://docs.filecoin.io/how-to/networks/#testnet). A preview of Space Race standings is available on the [Dashboard](https://spacerace.filecoin.io/). 
```

Comment:
This paragraph was originally for the calibration period. Now that it's over, let's delete this para entirely and move the dashboard link to the competition section.

```suggestion
```